### DISCUSSION TITLE
Scan Python libraries

### DISCUSSION THREAD

**Discussion by MosfetN** [glpi-project/glpi-agent#693] [Ideas]
Scan Python libraries

Hello,

Is there a way to retrieve all Python libraries (maybe this could be useful for VS Code also) ?
This could be a great improvement to check malicious ones and versions.
I very great improvement would be to compare those inputs with a CVE database...I know this is a huge change; it's just a dream :-)

**Comment by g-bougard** [comment]
Hi @MosfetN 

sorry, the purpose of glpi-agent is not to become a security issue scanner.

Anyway, you can check Collect task to look for files or folders installed on the computer: https://faq.teclib.com/03_knowledgebase/inventory/Collect_data/

**Reply by MosfetN** [reply]
Thanks Guillaume
I know it's not the purpose but it's already very useful to get version of installed software.
An enhancement would be to go deeper in this inventories.

Anyway,
Thanks a lot for all the work you make.
